flujo/cli/main.py

```
@@ -2294,10 +2311,23 @@ def _location(f: Any) -> dict[str, Any]:
             typer.echo(json.dumps(sarif))
         else:
             if report.errors:
-                typer.echo("[red]Validation errors detected:")
-                typer.echo(
-                    "[red]See docs: https://aandresalvarez.github.io/flujo/reference/validation_rules/"
-                )
+                try:
+                    from rich.console import Console as _Console
+
+                    _Console().print("[red]Validation errors detected[/red]:")
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)